src/treetensornetworks/projttns/projttnsum.jl

```diff
@@ -1,18 +1,25 @@
 """
 ProjTTNSum
 """
-struct ProjTTNSum{V} <: AbstractProjTTN{V}
-  terms::Vector{T} where {T<:AbstractProjTTN{V}}
-  function ProjTTNSum(terms::Vector{T}) where {V,T<:AbstractProjTTN{V}}
-    return new{V}(terms)
+struct ProjTTNSum{V,T<:AbstractProjTTN{V},Z<:Number} <: AbstractProjTTN{V}
+  terms::Vector{T}
+  factors::Vector{Z}
+  function ProjTTNSum(terms::Vector{<:AbstractProjTTN}, factors::Vector{<:Number})
+    return new{vertextype(eltype(terms)),eltype(terms),eltype(factors)}(terms, factors)
   end
 end
 
 terms(P::ProjTTNSum) = P.terms
+factors(P::ProjTTNSum) = P.factors
 
 copy(P::ProjTTNSum) = ProjTTNSum(copy.(terms(P)))
 
-ProjTTNSum(operators::Vector{<:AbstractTTN}) = ProjTTNSum(ProjTTN.(operators))
+function ProjTTNSum(operators::Vector{<:AbstractProjTTN})
+  return ProjTTNSum(operators, fill(1, length(operators)))
```

Comment:
```suggestion
  return ProjTTNSum(operators, fill(one(Bool), length(operators)))
```
to be more agnostic about the element type.